sdk/core/Azure.Core/src/Http/HttpRange.cs

```diff
@@ -0,0 +1,64 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System;
```

Comment:
nit: remove & sort usings

Reply:
it is both sorted and trimmed. at least this this is what VS thinks. Are you seeing anything specific that you think is not sorted/trimmed?

Reply:
Interesting VS sorting 😄